src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -31,37 +32,31 @@ def scan_for_suspect_projects() -> None:
 def _scan_for_suspect_projects() -> None:
     suspect_projects = set()
 
-    for project_id in realtime_metrics_store.projects():
+    for project_id in realtime_metrics.projects():
         suspect_projects.add(project_id)
         update_lpq_eligibility(project_id).apply_async()
 
     # Prune projects we definitely know shouldn't be in the queue any more.
     # `update_lpq_eligibility` should handle removing suspect projects from the list if it turns
     # out they need to be evicted.
-    current_lpq_projects = realtime_metrics_store.get_lpq_projects() or set()
+    current_lpq_projects = realtime_metrics.get_lpq_projects() or set()
     deleted_projects = current_lpq_projects.difference(suspect_projects)
     if len(deleted_projects) == 0:
         return
 
-    removed = realtime_metrics_store.remove_projects_from_lpq(deleted_projects)
+    realtime_metrics.remove_projects_from_lpq(deleted_projects)
 
-    if len(removed) > 0:
-        for project_id in removed:
-            logger.warning("Moved project out of symbolicator's low priority queue: %s", project_id)
-
-    not_removed = deleted_projects.difference(removed)
-    if len(not_removed) > 0:
-        logger.warning(
-            "Failed to move project(s) out of symbolicator's low priority queue: %s", not_removed
-        )
+    for project_id in deleted_projects:
+        # TODO: add metrics!
+        logger.warning("Moved project out of symbolicator's low priority queue: %s", project_id)
```

Comment:
This is the biggest logical change: we no longer try to figure out which projects may or may not have been removed (because the API doesn't allow us).  Instead we rely on the fact the API **will** succeed or will raise an exception.  So if we didn't die with an exception then this succeeded and all these projects were removed from the LPQ.

In theory a project could already have been removed before so it wasn't actually removed just now, but this really doesn't matter.  And in any case if that happens our tasks are not keeping up with real time.